content/docs/traffic-management/weighted-routes.md

```diff
@@ -0,0 +1,414 @@
+
+1. Deploy a hello-world app to demonstrate different backend responses.
+
+   ```yaml
+   kubectl apply -f- <<EOF
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     name: hello-world
+     namespace: httpbin
+   spec:
+     replicas: 1
+     selector:
+       matchLabels:
+         app: hello-world
+     template:
+       metadata:
+         labels:
+           app: hello-world
+       spec:
+         containers:
+         - name: hello-world
+           image: nginxdemos/hello:plain-text
+           ports:
+           - containerPort: 80
+   ---
+   apiVersion: v1
+   kind: Service
+   metadata:
+     name: hello-world
+     namespace: httpbin
+   spec:
+     selector:
+       app: hello-world
+     ports:
+     - port: 80
+       targetPort: 80
+   EOF
+   ```
+
+2. Update the HTTPRoute to demonstrate the default Gateway API precedence behavior by configuring two routes: `/anything` and `/anything/a`.
+
+   ```yaml
+   kubectl apply -f- <<EOF
+   apiVersion: gateway.networking.k8s.io/v1
+   kind: HTTPRoute
+   metadata:
+     name: httpbin
+     namespace: httpbin
+   spec:
+     parentRefs:
+       - name: http
+         namespace: {{< reuse "docs/snippets/namespace.md" >}}
+     hostnames:
+       - "www.example.com"
+     rules:
+       - matches:
+         - path:
+             type: PathPrefix
+             value: /anything
+         backendRefs:
+           - name: httpbin
+             port: 8000
+       - matches:
+         - path:
+             type: PathPrefix
+             value: /anything/a
+         backendRefs:
+           - name: hello-world
+             port: 80
+   EOF
+   ```
+
+3. Send a request to the `/anything/a` path. The request matches both the rules for `/anything` and `/anything/a`. Because the more specific `/anything/a` rule takes precedence, the request routes to the hello-world service instead of the httpbin service.
+   
+   {{< tabs items="Cloud Provider LoadBalancer,Port-forward for local testing" tabTotal="2" >}}
+   {{% tab tabName="Cloud Provider LoadBalancer" %}}
+   ```sh
+   curl -i http://$INGRESS_GW_ADDRESS:8080/anything/a -H "host: www.example.com:8080"
+   ```
+   {{% /tab %}}
+   {{% tab tabName="Port-forward for local testing"%}}
+   ```sh
+   curl -i localhost:8080/anything/a -H "host: www.example.com"
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+
+   Example output from the hello-world service:
+   
+   ```txt
+   HTTP/1.1 200 OK
+   server: envoy
+   date: Tue, 29 Jul 2025 20:07:36 GMT
+   content-type: text/plain
+   content-length: 167
+   expires: Tue, 29 Jul 2025 20:07:35 GMT
+   cache-control: no-cache
+   x-envoy-upstream-service-time: 0
+   
+   Server address: 10.244.0.8:80
+   Server name: hello-world-669dfbd799-g4jkg
+   Date: 29/Jul/2025:20:07:36 +0000
+   URI: /anything/a
+   Request ID: 796644ce8bda8ae5ecc36c5f4117a590
+   ```
+
+4. Send a request to the `/anything` path. This routes to the httpbin service because it only matches the less specific rule.
```

Comment:
```suggestion
4. Send a request to the `/anything` path. This example routes to the httpbin service because it only matches the less specific rule.
```